src/tool/subcommands/snapshot_cmd.rs

```diff
@@ -140,6 +160,42 @@ impl SnapshotCommands {
                 )
                 .await
             }
+            Self::Validate {
+                check_links,
+                check_network,
+                check_stateroots,
+                snapshot_files,
+                fail_fast,
+            } => {
+                let mut has_fail = false;
+                for file in snapshot_files {
+                    println!("Validating {}", file.display());
+                    let result = async {
+                        let store = AnyCar::try_from(file.as_path())?;
+                        validate_with_blockstore(
+                            store.heaviest_tipset()?,
+                            Arc::new(store),
+                            check_links,
+                            check_network.clone(),
+                            check_stateroots,
+                        )
+                        .await?;
+                        Ok::<(), anyhow::Error>(())
+                    }
+                    .await;
+                    if result.is_err() {
```

Comment:
It's usually useful to have the cause of the error, so I'd recommend printing it out as well.